### PR TITLE
Ticket 4.View a list of all articles with basic styling

### PR DESCRIPTION
**Pull request by greg19850**

### CODE REVIEW

File: src/components/SingleArticle.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function SingleArticle() {
```
Comment (HannahBurks) [inline src/components/SingleArticle.js]: Let's remove this component for now as it is not relevant to this ticket

File: src/utils/api.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+
+const articlesApi = axios.create({
+  baseURL: 'https://nc-backend-project-nc-news.onrender.com/api'
+});
+
+export const getCategories = () => {
```
Comment (HannahBurks) [inline src/utils/api.js]: Let's also remove this (regarding topics and categories for now - again not relevant to this ticket)

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Please remove all content unrelated to this ticket - once done, please send another PR and you should then be good to merge. Great work though Greg! Just make sure you only commit stuff up to the point of the ticket (nothing related to future tickets)

**Comment by cjplatten** [comment]
Looks good!

**Pull request merged**